### ISSUE TITLE
Backport to X version?

### ISSUE THREAD

**Issue by SHsuperCM** [SHsuperCM/CITResewn#120]
This issue is here to remind you that idgaf about old versions. They're too much to maintain and overall just not worth it.

Going forward this will be the issue stating which versions will be kept.

**Comment by SHsuperCM** [comment]
Unless there is a major issue with `v0.9.1` for 1.17, its support is dropped for now and `v1` will **not** be ported back.

As of this update, the only minecraft versions supported for `v1` are 1.18, 1.18.1 and 1.18.2.
1.18 will keep support until it becomes too much hassle to maintain after 1.19 is out.

**Comment by SHsuperCM** [comment]
1.19.0 is working, although with a bug that slows down loading a bit, the issue appears to be fixed in 1.19.1.
Seems as if 1.19.1 is going to be working with the current 1.19 versions of CIT Resewn(it has been working through the snapshots).

1.19.x is now the main development version and updates will be ported back to 1.18 for the foreseeable future.

**Comment by K4sum1** [comment]
It would genuinely be easier to get CIT Resewn working on 1.16.5 than to convert a CIT Resource pack to Chime.

**Comment by SHsuperCM** [comment]
> Just wanted to check, I assume you've moved on to 19.3 or even 19.4 so I'm guessing the trident/spyglass bug is there to stay on 19.2? It would be nice to confirm which versions you plan on supporting for now.

Development is still halted for a bit sadly but the intention is to update 1.19.2 as well since 1.19.3 is no longer related to 1.19 anymore(code wise)

**Comment by WeebKun1000** [comment]
I need a CIT reader for a 1.16.5 resource pack but I use sodium and many other fabric mods that don't allow me to use optifabric or any of its variations, please if you know any mod that can do that for 1.16.5 reply to this or please if you could quickly make a super simple one I don't need any new CIT marking item marking method I just want the classic to work because I cant bother having under 100 FPS on optifine opposed to 300 on sodium

**Comment by helldoodle-dev** [comment]
Make please for 1.16.5 ;(

**Comment by SHsuperCM** [comment]
Please read the latest post on the main mod issue:
https://github.com/SHsuperCM/CITResewn/issues/94#issuecomment-1576741913

TLDR - The plan is to switch to a multiversion build structure. This makes stuff easier to maintain and opens back up the possibility of backports.

**Comment by breadzzz08** [comment]
When 1.16.5?

**Comment by SHsuperCM** [comment]
Current version support status:
- Apologies for the long delay, `1.20.4` is now supported and is the main development version.
- Current supported versions are `1.19.4`, `1.20.1` and `1.20.4`.
- I am examining the possibility of `1.19.2` support.
- Versions earlier than `1.19.2` are not planned for the foreseeable future.


As for `1.20.5`, I am unsure of how to proceed yet with the changes made by the new item component system. I'll know more once it releases.

**Comment by FijiAura** [comment]
i swear though, 1.12.2 will make your computer explode due to people not porting optimisation mods to 1.12.2 as "its already optimised" like mojang just completely removed all optimisation in 1.13

**Comment by Mineblame16** [comment]
1.21.4 ? cit stuff aint fully supported yet til 1.21.5 , please release it
